src/components/ComposedModal/_composed-modal.scss

```
@@ -0,0 +1,5 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--composed-modal__body--small-margin-bottom {
+  margin-bottom: 1em;
```

This is probably a good one to use the tokens/variables for because it's directly related to the size of the Carbon component
```suggestion
  margin-bottom: $spacing-05;
```

---

@tay1orjones Thanks! Suggestion committed and we are ready to merge if you approve.